pkg/resource/aws/aws_route53_zone_test.go

```diff
@@ -12,11 +12,19 @@ func TestAwsRoute53Zone_String(t *testing.T) {
 		zone AwsRoute53Zone
 		want string
 	}{
-		{name: "",
+		{name: "test route53 zone stringer with name and id",
 			zone: AwsRoute53Zone{
 				Name: aws.String("example.com"),
+				Id:   "ZOS30SFDAFTU9__github-challenge-cloudskiff.cloudskiff.com_TXT",
```

Comment:
This is a full record ID ! This is very confusing, you should set a real zone ID instead : only `ZOS30SFDAFTU9`

Reply:
Ok my bad, it's because res.TerraformId() give me this.

Reply:
`TerraformId()` give you this for a `AwsRoute53Record` resource not for a `AwsRoute53Zone`, you should be clear about theses two resources differences

Reply:
My research show that it's only the field in the test file which look incorrect, so i let the String() method of route53_zone unchanged.